cves/CVE-2016-6817.yml

```diff
@@ -101,9 +102,9 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: false
```

Comment:
What are your thoughts about the developer not adding any automated tests? Based on the description, this seems like an easy vulnerability to add a unit test.

Reply:
I'm assuming the author thought the change was insignificant enough not to warrant tests, however I definitely agree that a quick unit test or two would greatly benefit the change